component-library/component-lib/src/lib/shared/breadcrumb/breadcrumb.component.ts

```diff
@@ -29,23 +36,67 @@ export interface IBreadcrumbConfig {
   selector: 'ircc-cl-lib-breadcrumb',
   templateUrl: './breadcrumb.component.html'
 })
-export class BreadcrumbComponent implements OnInit, OnChanges {
+export class BreadcrumbComponent implements OnInit, OnChanges, AfterViewInit {
   @Input() config: IBreadcrumbConfig = {
     id: '',
     baseUrlKey: '',
     type: 'href'
   };
   baseUrl = '';
+  separatorIcon: IIconButtonComponentConfig = {
+    id: 'breadcrumb_separator',
+    category: 'custom',
+    size: this.config?.size,
+    icon: {
+      class: 'fa-solid fa-ellipsis',
+      color: 'var(--text-primary)'
+    }
+  };
+  overflowLinks?: ILinkComponentConfig[];
+  normalLinks?: ILinkComponentConfig[]; // Links that are not overflow
+  displayOverflow = false;
+  private maxHeight: number = 0; // Max height of element in px
+  @ViewChild('breadcrumb_child') childRef?: ElementRef<HTMLParagraphElement>;
+  isChildOverflow: boolean = false;
   constructor(
     private translate: TranslateService,
-    private standalone: StandAloneFunctions
+    private standalone: StandAloneFunctions,
+    private el: ElementRef,
+    private renderer: Renderer2,
+    private changeRef: ChangeDetectorRef
   ) {}
 
   ngOnInit() {
     this.createLinks();
+    this.separatorIcon.size = this.config.size;
+    this.maxHeight = this.getMaxHeight();
+  }
+
+  ngAfterViewInit() {
+    setTimeout(() => {
+      this.createOverflows();
+      this.isChildOverflow = this.getChildOverflow();
+      this.changeRef.detectChanges();
+    }, 500);
```

Comment:
Can we move this into another hook to avoid having to setTimout?

Reply:
I have tried `ngAfterViewChecked` and it didn't work.